src/robotoff.js

```diff
@@ -126,5 +126,17 @@ export default {
     return axios.get(
       `${ROBOTOFF_API_URL}/images/logos?logo_ids=${logoIds.join(",")}`
     )
+  },
+
+  getNutritionValueFromImage(imageUrl) {
```

Comment:
```suggestion
  getNutritionValueFromImage(imageOcrUrl) {
```

You need a link to a json file containing the url